<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="Calc">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>575</width>
      <height>365</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(245, 245, 245)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="CalcPage">
     <widget class="QPushButton" name="Button_atan">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_equal">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>345</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
padding: 10px;
}</string>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_close">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_cos">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_del">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_clear">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(255, 204, 204);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_div">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_dot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_open">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_ln">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_sin">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Value">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>265</y>
        <width>65</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_depos">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>Depos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_sqrt">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_scale">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_acos">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_add">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_sub">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_log">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
     <widget class="QLabel" name="Sign">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/images/resource/sign.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_0">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_mod">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_mul">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_tan">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_x">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>220</y>
        <width>65</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>GrapX</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_asin">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_credit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>290</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>Loan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>65</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="GraphPage">
     <property name="styleSheet">
      <string notr="true">background-color: #FFF;</string>
     </property>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>285</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Graph">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>305</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
padding: 10px;
}</string>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Xmin">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>300</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>X min:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QLabel" name="Xmax">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>330</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>X max:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Back">
      <property name="geometry">
       <rect>
        <x>505</x>
        <y>295</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Xmin_edit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>300</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>-10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="Xmax_edit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>330</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Grapx">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>295</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid rgb(36, 48, 94);
border-radius: 10px;
background-color: rgb(236, 240, 249);
color: rgb(36, 48, 94);
}</string>
      </property>
      <property name="text">
       <string>GrapX</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
